### PR TITLE
test: (e2e) dynamic page layouts

### PR DESCRIPTION
**Pull request by AntonOlkhovskyi**
#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
- [x] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist

Documentation checklist:
- [x] update `README.md`
- [x] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

File: e2e/wdio/platform/pages/dynamic-page-layout.po.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import { BaseComponentPo } from './base-component.po';
+import { waitForElDisplayed, waitForPresent } from '../../driver/wdio';
+
+export class DynamicPageLayoutPo extends BaseComponentPo {
+    private url = '/dynamic-page';
+    root = '#page-content';
```
Comment (Seamoo13) [inline e2e/wdio/platform/pages/dynamic-page-layout.po.ts:6]: can remove root since you aren't using it

**Review by Seamoo13 — APPROVED** [review]
LGTM, just remove unused selector

File: e2e/wdio/platform/fixtures/appData/dynamic-page-layout-contents.ts [hunk 2]
```diff
@@ -0,0 +1,3 @@
+const main_button_title = 'Click to open full screen';
```
Comment (nikvalor) [inline e2e/wdio/platform/fixtures/appData/dynamic-page-layout-contents.ts:1]: export const main_button_title = 'Click to open full screen';

Comment (valorkin) [reply e2e/wdio/platform/fixtures/appData/dynamic-page-layout-contents.ts:1]: :+1:

**Review by nikvalor — APPROVED** [review]

**Pull request merged**